certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -75,10 +76,6 @@ func (p *DeployableByOlmCheck) operatorMetadata(bundleRef certification.ImageRef
 	}
 
 	catalogImage := viper.GetString(indexImageKey)
-	if len(catalogImage) == 0 {
```

Comment:
Is this validation no longer needed? Just want to make sure it wasn't removed on accident, as I don't see any validation for the new secrete code.

Reply:
I moved this to check_operator to ensure the check is failing fast.

Reply:
That makes sense, thanks for clarifying!